### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by AlexandraKlein**
Adds (mostly) CSS Slider (aka Finite ✌️Carousel✌️)
Adds `useMeasure` hook.

### CODE REVIEW

File: packages/slider/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    }, [containerWidth, trackWidth]);
+
+    React.useEffect(() => {
+        const container = containerRef.current;
+
+        if (!container) {
+            return;
+        }
+
+        containerRef.current.addEventListener('scroll', handleContainerScroll);
```
Comment (claus) [inline packages/slider/src/index.tsx:92]:
```suggestion
        container.addEventListener('scroll', handleContainerScroll);
```

Comment (AlexandraKlein) [reply packages/slider/src/index.tsx:93]: Good suggestion

File: packages/slider/src/Slider.module.scss [hunk 2]
```diff
@@ -2,10 +2,9 @@
     --slider-scrollbar-color: #b8b8b8;
     --slider-scrollbar-border-width: 0.35rem;
     --slider-scrollbar-border-radius: 2rem;
-    --slider-track-justify-content: flex-start;
-    --slider-slide-spacing: 1rem;
-    --slider-scroll-snap-type: x mandatory;
-    --slider-scroll-snap-align: center;
+    --slider-slide-spacing: 0;
```
Comment (claus) [inline packages/slider/src/Slider.module.scss:5]: Maybe `--slider-gap` would be a better name?

**Review by claus — APPROVED** [review]
Good!

File: packages/slider/src/Slider.module.scss (hunk 2 above)
Comment (AlexandraKlein) [reply packages/slider/src/Slider.module.scss:5]: Much better

**Comment by AlexandraKlein** [comment]
> Good!

Good N8!

**Comment by claus** [comment]
> > Good!
> 
> Good N8!

Not yet

**Pull request merged**